chain/chain/src/store.rs

```diff
@@ -2017,6 +2028,11 @@ impl<'a> ChainStoreUpdate<'a> {
         for key in stored_state_changes {
             self.gc_col(ColStateChanges, &key);
         }
+        // 3h. Delete from ColBlockRefCount
+        store_update.delete(ColBlockRefCount, block_hash_ref);
+        self.chain_store.block_refcounts.cache_remove(&block_hash.into());
```

Comment:
remove these lines - we already do `self.gc_col(ColBlockRefCount, &block_hash_vec)` below